java/client/src/main/java/glide/api/models/configuration/BaseClientConfiguration.java

```diff
@@ -51,4 +52,11 @@ public abstract class BaseClientConfiguration {
      * it will result in a timeout error. If not set, a default value will be used.
      */
     private final Integer requestTimeout;
+
+    /**
+     * Advanced users can pass an extended {@link glide.connectors.resources.ThreadPoolResource} to
```

Comment:
nit: you can shorten the link by adding import
```suggestion
     * Advanced users can pass an extended {@link ThreadPoolResource} to
```